doc/sphinx/run.rst

```diff
@@ -171,157 +171,219 @@ The equilibrium constant of the reaction is then given as
+.. pH 7). Further the :math:`pK_w=14` should be reproduced -also in the
+.. case of an initial excess of acid or base in the simulation box. Note
+.. that this only works for big enough volumes.
+
+
+
+
+Wang-Landau Reaction Ensemble 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. .. note:: Requires support for energy calculations for all used interactions since it uses Monte-Carlo moves which use energies in one way or the other.
+
+Combination of the Reaction Ensemble with the Wang-Landau algorithm 
+:cite:`wang01a`
+allows for enhanced sampling of the reacting system, and
+and for the determination of the density of states with respect 
+to the reaction coordinate or with respect to some other collective
+variable :cite:`landsgesell16a`. Here the 1/t Wang-Landau
+algorithm :cite:`belardinelli07a` is implemented since it
 does not suffer from systematic errors. Additionally to the above
 commands for the reaction ensemble use the following commands for the
 Wang-Landau reaction ensemble. For a description of the available methods see :mod:`espressomd.reaction_ensemble`:
 
-Constant pH method
-------------------
-.. note:: Requires support for energy calculations for all used interactions since it uses Monte-Carlo moves which use energies.
+Constant pH simulation using the Reaction Ensemble
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. .. note:: Requires support for energy calculations for all used interactions since it uses Monte-Carlo moves which use energies.
 
 In the constant pH method due to Reed and Reed
-::cite:`reed92a` it is possible to set the chemical potential
-of :math:`H^{+}` ions, assuming the simulated system is coupled to an
+:cite:`reed92a` it is possible to set the chemical potential
+of :math:`H^{+}` ions, assuming that the simulated system is coupled to an
 infinite reservoir. This value is the used to simulate dissociation
 equilibrium of acids and bases. Under certain conditions, the constant
-pH method can yield equivalent results as the reaction ensemble. For
-more information see ::cite:`landsgesell16b`. However, it
+pH method can yield equivalent results as the reaction ensemble :cite:`landsgesell16b`. However, it
 treats the chemical potential of :math:`H^{+}` ions and their actual
 number in the simulation box as independent variables, which can lead to
-serious artifacts. The constant pH method significantly differs in its
-derivation compared to the Reaction Ensemble. The constant pH method can
-be used by initializing the reactions of interest with the commands
-mentioned for the reaction ensemble. However with the difference that
+serious artifacts. 
+The constant pH method can be used within the reaction ensemble module by
+initializing the reactions with the standard commands of the reaction ensemble. 
+
+However with the difference that
 you do not provide the dimensionless reaction constant but directly the
 *apparent reaction constant* (from the law of mass action) :math:`K_a`
```

Comment:
can you pleas write K_c instead of K_a, after you introduced it above.

Reply:
I forgot to change this one. However, this note is a consequence of the previous misunderstanding. Now, when we both agree that RE uses the dimensional equilibrium constant, this note should be dropped. But we need to define the reaction which is implicitly used in the constant pH method.